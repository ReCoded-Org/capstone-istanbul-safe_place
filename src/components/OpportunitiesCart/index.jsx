import React from "react";
import { Col } from "react-bootstrap";
import { useTranslation } from "react-i18next";
import "./index.scss";
import { locationSvg } from "../../components/SeekHelpCarousel/images/map-marker.svg";

export default function OpportunitiesCart(opportunity) {
  const { t } = useTranslation();
  return (
    <Col className="opportunities">
      <div className="opportunityPostDate">{t(opportunity.date)}</div>
      <div className="opportunityTitle">{t(opportunity.title)}</div>
      <div className="opportunityDescription">{t(opportunity.description)}</div>
      <span className="borderLine"></span>
      <div className="opportunityContactsDetails">
        <div className="opportunityAddress">{t(opportunity.address)}</div>
        <div className="opportunityPhoneNumber">
          {t(opportunity.phoneNumber)}
        </div>
      </div>
    </Col>
  );
}
