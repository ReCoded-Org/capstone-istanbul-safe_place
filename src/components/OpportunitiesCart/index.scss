@import "../../style/variables"; 

.opportunities{
width: 356px;
height: 283px;
background: #FFFEFC;
padding:20px;
margin: 20px;
}

.opportunityPostDate {
padding-left: 180px;
width: 70px;
height: 17px;
}