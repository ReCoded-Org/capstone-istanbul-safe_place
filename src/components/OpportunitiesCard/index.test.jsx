import React from "react";
import renderer from "react-test-renderer";
import OpportunitiesCart from "./index";

it("renders correctly and matches the snapshot", () => {
  const tree = renderer.create(<OpportunitiesCard />).toJSON();
  expect(tree).toMatchSnapshot();
});
