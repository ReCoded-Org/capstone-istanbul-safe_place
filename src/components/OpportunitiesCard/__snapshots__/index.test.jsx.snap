// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly and matches the snapshot 1`] = `
<div
  className="opportunities col"
>
  <div
    className="postDate"
  />
  <div
    className="title"
  />
  <div
    className="description"
  />
  <span
    className="borderLine"
  />
  <div
    className="contactsDetails"
  >
    <div
      className="address"
    />
    <div
      className="nameOfOpportunityOwner"
    />
    <div
      className="phoneNumber"
    />
  </div>
</div>
`;
